static/css/app.css

```diff
@@ -1,427 +0,0 @@
-/*.is-accordion-submenu-parent a*/ [data-accordion-menu] li a {
```

Comment:
Why is all this custom css gone?

Reply:
Oops. Didn't notice the diff that the custom css was missing. My bad